addForm.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once 'dbConnect.php';
+$db = dbConnect();
+addSet($db);
+
+function addSet($db){
+
+    if(isset($_POST["name"], $_POST["number"], $_POST["pieces"], $_POST["year"])) {
```

Comment:
Single quotes can be used here as best practice (single quotes where possible, double quotes where needed for interpolation)